grisha.pletin/SixHW/HTMLparse.rb

```diff
@@ -0,0 +1,36 @@
+require 'rubygems'
```

Comment:
Please define one class in a one file. Also in ruby we name our .rb file the same as class that is defined in the file.